src/components/Link/Link.tsx

```diff
@@ -107,7 +107,9 @@ export const Link = chakra(
     let variant = "link";
 
     if (typeof children === "string" && !href) {
-      throw new Error("`Link` needs the `href` prop.");
+      throw new Error(
+        "NYPL Reservoir Link: The `Link` component needs the `href` prop if its child element is a string."
```

Comment:
Interesting that we missed these during the push to v1.

Reply:
I think because these are actual errors and not warnings.